libs/agno/agno/agent/agent.py

```diff
@@ -3808,3 +3808,97 @@ def cli_app(
                 break
 
             self.print_response(message=message, stream=stream, markdown=markdown, **kwargs)
+
+    async def acli_app(
```

Comment:
If we are doing this it might be worthit to consider the DX here. 
1. We could consider making CLI / Live a separate class that takes an agent in a constructor, similar to a playground app setup. I kinda feel like it would be more intuitive to use, even though it is more lines of code? 
2. Alternatively this needs a better name I think. `run_live` and `arun_live`?